### DISCUSSION TITLE
What is the method to add organization field to response content of /api/userinfo

### DISCUSSION THREAD

**Discussion by robincodex** [casdoor/casdoor#1923] [Q&A] [answered]
Hi, I have a [Harbor](https://goharbor.io/) service where I login to my harbor through OIDC, but /api/userinfo did not return the organization to which the user belongs. How should I add it?

**Comment by hsluoyz** [comment, marked as answer]
@robincodex added in: https://github.com/casdoor/casdoor/commit/06543a01d34d73db6e7ea898aa7cc0c461f142e0 , please wait for next release
